src/components/Image/Image.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import ImageWET from '@components/Image';
+
+describe('Image Tests', () => {
+  test('Test image component classes', () => {
+    render(
+      <ImageWET
+        src="https://media.istockphoto.com/photos/very-closeup-view-of-amazing-domestic-pet-in-mirror-round-fashion-is-picture-id1281804798?b=1&k=20&m=1281804798&s=170667a&w=0&h=HIWbeaP_cQSngCz7l9t3xwyE2eyzVgIy3K6xIqPhJQA="
+        roundedCircle
+        thumbnail
+        rounded
+        responsive
+        fullWidth
+        media
+        thumbnailTile
+      />
+    );
+    expect(screen.getByRole('img')).toHaveClass('img-circle');
+    expect(screen.getByRole('img')).toHaveClass('img-thumbnail');
```

Comment:
I would test each of these separately but group them under the same describe. As an example of nested describe statements you can check out the tests for the button component (line 16)